### PR TITLE
[New post] How to build GenAI application using Amazon Bedrock with AWS PrivateLink to protect your data privacy

### PR DESCRIPTION
**Pull request by Xiaoyuao988**
# Content Pull Request Checklist

⚠️⚠️⚠️⚠️⚠️
Before creating this PR, if you are authoring a blog / opinion piece, please do not create a PR as we have released the new Post Editor allowing authoring directly in your browser on community.aws. Please see [this post](https://community.aws/content/2YmrbVIkokIRiRtjDFTUWbAdcBi/ready-set-post-community-articles-are-live-on-community-aws) with all the details on how to get started publishing your content!
⚠️⚠️⚠️⚠️⚠️

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [X ] The title of this pull request reflects the title of my content.
- [ X] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [ X] This pull request contains one piece of content (post or page).

**Important: if you are submitting a PR to update existing content, please ensure you [sync'ed your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) first before you created your branch.**

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](https://github.com/build-on-aws/content/blob/main/LICENSE). Notwithstanding the above, if you provide source code text within your content piece, you are providing such source code under the [MIT-0](https://opensource.org/license/mit-0/) license.**

### CODE REVIEW

**Comment by cherbk** [comment]
published using post editor

File: concepts/build-GenAI-app-Bedrock-privateLink/index.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+authorGithubAlias: Xiaoyuao988
+authorName: Xiaoyu Xing
+date: 2023-12-20
+---
+
+Most would likely agree that **Data Privacy** is an important issue that cannot be overlooked when it comes to data analytics and machine learning. **Generative AI** is making waves, but without proper security control, Generative AI tools could be susceptible to data theft. This could lead to unauthorized access or disclosure of sensitive information, potentially leading to privacy violations and possible misuse of both sensitive personal data or intellectual property.
+
+[Amazon Bedrock](https://aws.amazon.com/bedrock/security-compliance/), a fully managed service, allows you to build generative AI applications that support data security and compliance standards, including **GDPR** and **HIPAA**.
+
+With **Amazon Bedrock**, ***(not limited to)***
```
Comment (DalM2023) [inline concepts/build-GenAI-app-Bedrock-privateLink/index.md:18]: Please consider rewriting this sentence. It's incomplete. What are you trying to say here? 


```suggestion
With **Amazon Bedrock**, ***(not limited to)***
```

File: concepts/build-GenAI-app-Bedrock-privateLink/index.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
+---
+title: "How to build GenAI application using Amazon Bedrock with AWS PrivateLink to protect your data privacy"
```
Comment (DalM2023) [inline concepts/build-GenAI-app-Bedrock-privateLink/index.md:2]:
```suggestion
title: "Build GenAI Applications Using Amazon Bedrock With AWS PrivateLink To Protect Your Data Privacy"  
```

File: concepts/build-GenAI-app-Bedrock-privateLink/index.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+* Your data is always **encrypted in transit** (***TLS1.2***) and **at rest**. You can use your own keys or [AWS KMS (Key Management Service)](https://aws.amazon.com/kms/) keys to encrypt the data.
+* You can encrypt and store **customized (fine-tuned) models** with KMS key that either managed by AWS or by yourself.
+  You can further configure VPC for fine-tuning jobs with Bedrock, which prevent the **training data** from being accessible over the internet.
+* **Potential misuse** is prevented by [Bedrock abuse detection](https://docs.aws.amazon.com/bedrock/latest/userguide/abuse-detection.html). This feature **automatically** ***(no human review or access to user inputs or model outputs)*** identifies and mitigates potential violations of AWS's [Acceptable Use Policy (AUP)](https://aws.amazon.com/aup/) and [Responsible AI Policy](https://aws.amazon.com/machine-learning/responsible-ai/policy/), or a 3rd party model provider's AUP.
```
Comment (DalM2023) [inline concepts/build-GenAI-app-Bedrock-privateLink/index.md:24]:
```suggestion
* **Potential misuse** is prevented by [Bedrock abuse detection](https://docs.aws.amazon.com/bedrock/latest/userguide/abuse-detection.html). This feature **automatically** ***(no human review or access to user inputs or model outputs)*** identifies and mitigates potential violations of AWS's [Acceptable Use Policy (AUP)](https://aws.amazon.com/aup/) and [Responsible AI Policy](https://aws.amazon.com/machine-learning/responsible-ai/policy/), or a third party model provider's AUP.
```

File: concepts/build-GenAI-app-Bedrock-privateLink/index.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
+---
+title: "How to build GenAI application using Amazon Bedrock with AWS PrivateLink to protect your data privacy"  
+description: "This post is about **what** is **Private Link**, **Why** to use Private Link to protect Data Privacy, and **How** to build GenAI application using Amazon Bedrock with Private Link."
```
Comment (DalM2023) [inline concepts/build-GenAI-app-Bedrock-privateLink/index.md:3]: The descriptions need to be 60 characters or fewer. They can't repeat the messaging in the title. They should focus on what solution the reader gets from reading the piece. I wrote an example below but feel free to rewrite.
 
```suggestion
description: "Manage and protect data security and compliance standards."
```

File: concepts/build-GenAI-app-Bedrock-privateLink/index.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+
+[Amazon Bedrock](https://aws.amazon.com/bedrock/security-compliance/), a fully managed service, allows you to build generative AI applications that support data security and compliance standards, including **GDPR** and **HIPAA**.
+
+With **Amazon Bedrock**, ***(not limited to)***
+* Your **content** (e.g. prompt data, fine-tuning data, vector store data with RAG (***Retrieval Augmented Generation***) etc.) is not shared with 3rd party model providers (including AWS).
+* You can use **AWS PrivateLink** to establish private connectivity between the **Foundation Models (FM)** and on-premises networks or your **AWS VPC (Virtual Private Cloud)**, without exposing your traffic to the **public internet**. (***This blog post's focus***)
+* Your data is always **encrypted in transit** (***TLS1.2***) and **at rest**. You can use your own keys or [AWS KMS (Key Management Service)](https://aws.amazon.com/kms/) keys to encrypt the data.
+* You can encrypt and store **customized (fine-tuned) models** with KMS key that either managed by AWS or by yourself.
```
Comment (DalM2023) [inline concepts/build-GenAI-app-Bedrock-privateLink/index.md:22]:
```suggestion
* You can encrypt and store **customized (fine-tuned) models** with a KMS key that's either managed by AWS or by yourself.
```

File: concepts/build-GenAI-app-Bedrock-privateLink/index.md [hunk 6]
```diff
@@ -0,0 +1,209 @@
+---
```
Comment (DalM2023) [inline concepts/build-GenAI-app-Bedrock-privateLink/index.md:1]: Hello,

The blog has good structure and a strong intro that highlights the reasons why someone would need to read the blog. I made edits on grammar, titles and left a few notes where I got confused. Please see below for more information. 

Overall, you have a good blog here. Well done on a first draft!

**Review by DalM2023 — COMMENTED** [review]
1st round edit done. Author revision requested.

**Comment by cherbk** [comment]
@Xiaoyuao988 The first round of editorial review is finished. Kindly review and incorporate the suggested changes in the PR. Thank you!

File: concepts/build-GenAI-app-Bedrock-privateLink/index.md (hunk 1 above)
Comment (Xiaoyuao988) [reply concepts/build-GenAI-app-Bedrock-privateLink/index.md:18]: What I mean is that I haven't listed all the benefits.
Re-write by removing "***(not limited to)***", and adding sentence: "As you can see, Amazon Bedrock provides you comprehensive security and compliance capabilities. **Here I just list some of them**."

**Review by cherbk — APPROVED** [review]

**Review by cherbk — APPROVED** [review]

**Pull request merged**